internal/observability/event/eventer.go

```diff
@@ -616,7 +668,7 @@ func (e *Eventer) writeSysEvent(ctx context.Context, event *sysEvent) error {
 }
 
 // writeAudit writes/send an audit event
-func (e *Eventer) writeAudit(ctx context.Context, event *audit) error {
+func (e *Eventer) writeAudit(ctx context.Context, event *audit, opt ...Option) error {
```

Comment:
you could use `_` instead of `opt` so it's clear that currenlty no options are supported.

Reply:
Done